### DISCUSSION TITLE
gui.bat crashing on startup

### DISCUSSION THREAD

**Discussion by Aramis-Colon** [bmaltais/kohya_ss#542] [Q&A]
Just did an install of stable diffusion and kohya on my desktop and when I open try to launch the gui.bat it immediately crashes. This is the error that I am receiving. 

![image](https://user-images.githubusercontent.com/72364493/229872139-96b96445-fb2c-4663-b65d-6d6acaa8f656.png)

I am able to use kohya fine on my laptop. I went through the same process to do the install on the desktop (windows). I tried doing a clean install after the error just in case, but still no luck. Any suggestions?

**Comment by jstayco** [comment]
You can try opening requirements.txt and commenting out the line for lycoris git with a #. Then uncomment the line that says lycoris-core==0.1.4.

**Reply by Aramis-Colon** [reply]
Thanks for the suggestion. I tried, but unfortunately it still crashes and this time an error doesn't even show up. The command window will appear for a second and then immediately close.
